package.json

```diff
@@ -90,7 +92,8 @@
     "sockjs": "0.3.x",
     "sockjs-client": "1.0.x",
     "through2": "2.0.x",
-    "ws": "1.0.x"
+    "ws": "1.0.x",
+    "lws": ">=0.6.1"
```

Comment:
I would just use the same semver pattern as we are doing for the rest of the modules `0.6.x`